systemtest/src/main/java/io/strimzi/systemtest/cli/KafkaCmdClient.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.cli;
+
+import io.strimzi.api.kafka.model.KafkaResources;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static io.strimzi.test.k8s.KubeClusterResource.cmdKubeClient;
+
+public class KafkaCmdClient {
```

Comment:
Why is this not some `KafkaUtils` class? I'm not entirely sure how do you distinguish these and decie which should be which.

Reply:
The purpose of `KafkaCmdClient `is to have mainly encapsulated kafka.sh scripts (Kafka CLI). On the other hand, KafkaUtils is for the waits for concrete resource, verification via kubeClient and so on. What do you think @Frawless ?